pysatCDAAC/tests/test_instruments.py

```diff
@@ -67,9 +76,9 @@ def test_altitude_bin_keyword(self, inst_dict, bin_num):
         rem = np.remainder(self.test_inst['MSL_bin_alt'].values, 5)
         idx, idy, = np.where(rem == 0)
         idx2, idy2, = np.where(np.isnan(rem))
-        assert len(idx) + len(idx2) == np.product(rem.shape)
+        assert len(idx) + len(idx2) == np.prod(rem.shape)
 
         # Confirm length of each profile corresponds to bin_num
-        assert self.test_inst.data.dims['RO'] == bin_num
+        assert self.test_inst.data.sizes['RO'] == bin_num
```

Comment:
Is this change very sensitive to xarray versions?

Reply:
First added in v0.9.0 (2017).